### PR TITLE
Dar Profile Created

### PR DESCRIPTION
**Pull request by darwright33**

### CODE REVIEW

File: dar_profile.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Its the Dar show!
+
+Favorite City: LAS VEGAS!
+Favorite Food: Breakfast
+Favorite color: cerulean blue
+Favorite animal: cat
```
Comment (kselvocki) [inline dar_profile.md:6]: What are your cats' names?

**Review by kselvocki — CHANGES_REQUESTED** [review]

**Review by mrmorris — DISMISSED** [review]
Looks good, but you should take care of your teammates change request first...

**Review by kselvocki — APPROVED** [review]

File: dar_profile.md (hunk 1 above)
Comment (darwright33) [reply dar_profile.md:6]: Updated!

File: dar_profile.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
```
Comment (m-kinkade) [inline dar_profile.md:6]: What is your favorite holiday?

File: dar_profile.md (hunk 1 above)
Comment (ravensisco) [reply dar_profile.md:6]: Looking Great!

**Review by mrmorris — APPROVED** [review]

**Pull request merged**